{"name": "Buscar Appium en Google", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\n\n", "trace": "  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/test_google_steps.py\", line 13, in step_search_google\n    search_box = context.driver.find_element(By.NAME, \"q\")\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/appium/webdriver/webdriver.py\", line 384, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "steps": [{"name": "Given que abro Brave en el dispositivo", "status": "passed", "start": 1756266506244, "stop": 1756266508249}, {"name": "When busco \"Appium\" en Google", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\n\n", "trace": "  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/test_google_steps.py\", line 13, in step_search_google\n    search_box = context.driver.find_element(By.NAME, \"q\")\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/appium/webdriver/webdriver.py\", line 384, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/Users/gianperalta/appium-bdd-brave/venv/lib/python3.13/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "start": 1756266508250, "stop": 1756266508731}, {"name": "Then debería ver resultados en la página", "status": "skipped", "start": 1756266508732, "stop": 1756266508732}], "start": 1756266500569, "stop": 1756266508732, "uuid": "196cbe47-6fe6-42de-980c-6517ac4ba9e5", "historyId": "ed572c78ccedc7f84009730c4fe26b91", "fullName": "Buscar en Google: Buscar Appium en Google", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Buscar en Google"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}